midterm-2023-11-06/code/5169-vladimir-andonov/scripts.js

```diff
@@ -0,0 +1,91 @@
+document.addEventListener('DOMContentLoaded', () => {
+  
+  fetch('https://raw.githubusercontent.com/sweko/uacs-internet-programming-exams/main/midterm-2023-11-06/data/bands.json')
```

Comment:
Loading data properly (it would have been nicer to use the `async / await` syntax, but this works too (not terribly happy about defining functions inside callbacks)
+10